backend/CircleForms/Controllers/ApiController.cs

```diff
@@ -7,7 +7,7 @@
 namespace CircleForms.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("/")]
     public class ApiController : ControllerBase
```

Comment:
Probably better to rename it to RootController with a comment, to better represent it's route

Reply:
I don't it's necessary to do anything about this controller at all since it's a placeholder that will get removed, actually. I will revert controller changes